samples/createNotebook.bal

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerinax/microsoft.onenote as onenote;
```

Comment:
Do we need `as onenote` alias?

Reply:
This is to make it readable.
Otherwise we have to add `microsoft.onenote` in all the places which will look uglier.

Reply:
Why? We can use `onenote` directly.

Reply:
Will check

Reply:
Done